src/Microsoft.AspNet.Http.Extensions/HeaderDictionaryExtensions.cs

```diff
@@ -23,7 +24,7 @@ public static void Append(this IHeaderDictionary headers, string key, StringValu
         /// </summary>
         /// <param name="key">The header name.</param>
         /// <param name="values">The header values.</param>
-        public static void AppendCommaSeparatedValues(this IHeaderDictionary headers, string key, params string[] values)
+        public static void AppendCommaSeparatedValues(this IDictionary<string, StringValues> headers, string key, params string[] values)
```

Comment:
Yep, I'm looking at the same PR, I just added a comment that being IDictionary is probably a good thing :)